### PR TITLE
Adjust recipe for supporting python 3.x

### PR DESCRIPTION
**Pull request by katietz**
* bump build number
* adjust broken script line
* add dependencies for setuptools, wheels
* add patch file to get rid of deprecated open with mode "U" usage, which is default nowadays

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -7,15 +7,22 @@ package:
...
+  build:
+    - patch     # [not win]
+    - m2-patch  # [win]
   host:
     - python
     - pip
+    - setuptools
+    - wheel
   run:
     - python
```
Comment (skupr-anaconda) [inline recipe/meta.yaml:28]: The license is `BSD-2-Clause`

**Review by skupr-anaconda — COMMENTED** [review]
The package is pretty obsolete

File: recipe/meta.yaml (hunk 1 above)
Comment (katietz) [reply recipe/meta.yaml:28]: I added it and license_family ... and yes, it is pretty obsolete but for theano .... which is actual dead too

**Review by ELundby45 — APPROVED** [review]
LGTM. Thanks!

**Pull request merged**